alpha/scripts/utils/enableVirtualEventPermissionToAllPartners.php

```diff
@@ -4,16 +4,20 @@
  *
  *
  * Examples:
- * php enableVirtualEventPermissionToAllPartners.php
- * php enableVirtualEventPermissionToAllPartners.php realrun
+ * php enableVirtualEventPermissionToAllPartners.php 99 1000
+ * php enableVirtualEventPermissionToAllPartners.php 99 1000 realrun
  *
  * @package Deployment
  * @subpackage updates
  */
 
-$dryRun = true;
-if (in_array('realrun', $argv))
-	$dryRun = false;
+$dryRun = in_array('realrun', $argv) ? false : true;
```

Comment:
better move this line after the  ($argc < 3) check and check the exact argv value and not that one of the args is realrun